### PR TITLE
US018: User can edit name and phonenumber

### PR DESCRIPTION
**Pull request by dilmurod-rozikov**

### CODE REVIEW

**Comment by dilmurod-rozikov** [comment]
[-	US018 (1): User (client) can edit his name and phone number (use Identity area) Add Name and PhoneNumber fields to Registration form](https://trello.com/c/Ivi5GU7L/21-us018-1-user-client-can-edit-his-name-and-phone-number-use-identity-area-add-name-and-phonenumber-fields-to-registration-form)

File: ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml [hunk 1]
```diff
@@ -12,12 +12,12 @@
         <form id="profile-form" method="post">
             <div asp-validation-summary="ModelOnly" class="text-danger" role="alert"></div>
             <div class="form-floating mb-3">
-                <input asp-for="Username" class="form-control" placeholder="Please choose your username." disabled />
-                <label asp-for="Username" class="form-label"></label>
+                <input asp-for="Input.Name" class="form-control"/>
+                <label asp-for="Input.Name" class="form-label">Enter your name</label>
```
Comment (SamDotNetDev) [inline ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml:16]: Where is the UserName that shows Email? Why in Index are not you showing it

File: ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs [hunk 2]
```diff
@@ -98,18 +99,31 @@ public async Task<IActionResult> OnPostAsync()
...
                 {
                     StatusMessage = "Unexpected error when trying to set phone number.";
                     return RedirectToPage();
                 }
             }
 
+            if (Input.Name != user.Name)
+            {
+                user.Name = Input.Name;
+                user.UserName = Input.Name;
```
Comment (SamDotNetDev) [inline ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs]: Why after changing the Name are you editing the Username to Name?
UserName never changes. After registration UserName automatically gets Email.

**Review by SamDotNetDev — CHANGES_REQUESTED** [review]

File: ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml (hunk 1 above)
Comment (SamDotNetDev) [reply ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml:16]: ParkingZonePhoto

Comment (dilmurod-rozikov) [reply ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml:16]: image
So name and username both two different thing and username is email right?

Comment (SamDotNetDev) [reply ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml:16]: yeah right

**Review by SamDotNetDev — APPROVED** [review]

**Review by abdullokhtolibjonov — APPROVED** [review]

**Pull request merged**